Update dUNL facts & rippled's transfer (#1503)
XRPLF/xrpl-dev-portal: content/faq.md

For recommendations and best practices, see [Run `rippled` as a Validator](run-rippled-as-a-validator.html).

#### If the dUNL has the most influence on the network, then is the XRPL centralized?

> False, validators can choose to not listen to the dUNL or any widely used UNLs for that matter. They can can create a brand new UNL at anytime.

Reviewer: ```suggestion
Validators can choose to not use the dUNL, or any widely-used UNL for that matter. Anyone can create a new UNL at any time.
```

> There can be other UNLs in use on the network. The dUNL is the most widely-used UNL, but validators can choose others.

Reviewer 2: ```suggestion
There can be multiple UNLs in use on the same network. Each operator can customize their server's own UNL or choose to follow a different recommended list. All these servers can still operate on the same chain and reach consensus with one another.

However, if your UNL does not have sufficient overlap with the UNLs used by others, there is a risk that your server forks away from the rest of the network. As long as your UNL has > 90% overlap with the one used by people you're transacting with, you are completely safe from forking. If you have less overlap, you may still be able to follow the same chain, but the chances of forking increase with lower overlap, worse network connectivity, and the presence of unreliable or malicious validators on your UNL.
```

> 

Reviewer 3: I think we should mention something about the possibility of forking when nodes use different UNLs

> We're also not taking into account of the other UNLs on the network, which also has a huge say in ledger progress. The dUNL is simply the most widely used UNL out there, and validators can choose to opt-out and use another UNL.

Reviewer: ```suggestion
There can be other UNLs in use on the network. The dUNL is the most widely-used UNL, but validators can choose others.
```

No, Ripple does not own or control the XRP Ledger or XRP Ledger network.

> Ripple does contribute to a reference implementation of the core XRP Ledger server ([`rippled`](https://github.com/xrplf/rippled)) and employs a team of engineers who contribute to the open-source codebase. Ripple also periodically publishes pre-compiled binary packages of the software as a convenience. Anyone is free to [download and compile the software from source](install-rippled.html), if they prefer.

Reviewer: ```suggestion
Ripple contributes to a reference implementation of the core XRP Ledger server ([`rippled`](https://github.com/xrplf/rippled)) and employs a team of engineers who contribute to the open-source codebase. Ripple periodically publishes pre-compiled binary packages of the software for convenience. Anyone can [download and compile the software from source](install-rippled.html).
```

> Ripple is also one of several entities who publish recommended UNL settings and runs validators on the network. As of late-2022, Ripple runs 3 of the 35+ validators in the default UNL.

Reviewer: ```suggestion
Several entities publish recommended validator lists (UNLs). As of September 2022, Ripple runs 3 of the 35 validators in the default UNL.
```
